### PR TITLE
Wiki migration and link updates

### PR DESCRIPTION
**Pull request by Piinks**
Part of https://github.com/flutter/flutter/issues/145009

This migrates the flutter-intellij wiki into checked in files, and updates all old wiki links

I'll update the original wiki pages to point to these new places after this lands

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: docs/Dev-Channel.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+We have an early-access version of the plugin, called the dev channel. This version will be updated weekly, containing all changes made in the previous week. To use it you need to configure IntelliJ or Android Studio to look for plugins on the dev channel repository.
+
+NOTE: If you already have the Flutter plugin installed, please uninstall it before proceeding. Once you have the plugin installed from one channel it will not be available on other channels.
+
+Open the Plugins preference page (Mac) or settings page (Windows, Linux). Click the gear at the top-right and select _Manage Plugin Repositories_.
```
Comment (stuartmorgan) [inline docs/Dev-Channel.md:5]: Optional drive-by improvement: "Mac" should be "macOS"

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Frequently Asked Questions
+
+### How do I configure a Flutter SDK?
+
+The Flutter plugin for IntelliJ depends on a Flutter SDK to be configured in the Flutter Plugin (**Preferences>Languages & Frameworks>Flutter**). This should point to the Flutter install location.
+
+### How do I build the plugin?
+
+For details about developing with IntelliJ, see our
+[contributing](https://github.com/flutter/flutter-intellij/blob/master/CONTRIBUTING.md
```
Comment (stuartmorgan) [inline docs/FAQ.md:10]: Could this just be a relative link now?

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Welcome to the `flutter-intellij` wiki!
+
+This wiki is intended largely for development and process documentation.  
+
+For general plugin usage instructions see the [readme](https://github.com/flutter/flutter-intellij/blob/master/README.md). If you want to contribute please see the [contributor docs](https://github.com/flutter/flutter-intellij/blob/master/CONTRIBUTING.md).
```
Comment (stuartmorgan) [inline docs/README.md:5]: Relative links?

**Review by stuartmorgan — APPROVED** [review]
LGTM; all comments optional

**Review by jwren — APPROVED** [review]
